### PR TITLE
Pattern Library: Replace SVGs with PNGs in `PatternsGetStarted`

### PR DESCRIPTION
**Pull request by fredrikekelund**
Fixes https://github.com/Automattic/dotcom-forge/issues/6141

## Proposed Changes

The current images in `PatternsGetStarted` have been exported as SVGs from Figma (BnTfb6hzKcRTb2LBtuPwKe-fi-3%3A950), but they actually contain bitmaps. I've exported the images again from the highest-resolution versions I could find in Figma, resized and compressed them, and applied an unsharp mask to `understand-page-layouts.png` and `use-block-patterns.png` since those images were quite unsharp in their originals.

## Testing Instructions

1. Navigate to `/patterns`
2. Scroll down to the `All about patterns` section
3. Ensure that the images look good and uniform

### CODE REVIEW

File: client/my-sites/patterns/components/get-started/images/use-block-patterns.png
Comment (fredrikekelund) [inline client/my-sites/patterns/components/get-started/images/use-block-patterns.png:1]: The old image was 1.6 MB and this new one is ~100 kB

File: client/my-sites/patterns/components/get-started/images/understand-page-layouts.png
Comment (fredrikekelund) [inline client/my-sites/patterns/components/get-started/images/understand-page-layouts.png:1]: Just noting: the old image was 1.8 MB and this new one is ~115 kB

File: client/my-sites/patterns/components/get-started/images/preview-publish.png
Comment (fredrikekelund) [inline client/my-sites/patterns/components/get-started/images/preview-publish.png:1]: Just noting: the old image was ~744 kB and this new one is ~23 kB

File: client/my-sites/patterns/components/get-started/style.scss [hunk 1]
```diff
@@ -33,8 +33,12 @@
 	flex: 1;
 
 	.patterns-get-started__item-image {
+		aspect-ratio: 16 / 9;
+		border-radius: 4px;
+		margin-bottom: 16px;
+		object-fit: cover;
+		overflow: hidden;
```
Comment (nightnei) [inline client/my-sites/patterns/components/get-started/style.scss]: Don't see necessity, it seems they are redundant
```suggestion
		border-radius: 4px;
		margin-bottom: 16px;
```

**Review by nightnei — APPROVED** [review]
LGTM, left just small thing regarding styles

**Pull request merged**